README.md

```diff
@@ -3,12 +3,9 @@
 Let's walk through the steps of creating the app we've been working on above in
 a real world environment.
 
-## Download the Ext JS Framework
-
-If you have not already done so, download and unpack the Ext JS framework from
-either the Products section of the main Sencha website
-([www.sencha.com](http://www.sencha.com)) or from the downloads section of the
-Sencha Support portal.
+## Getting started
+### Prerequisite
+- Install [Node.js](https://nodejs.org/) (^6.9.2)
```

Comment:
Do we have an actual dependency on 6.9 or can we update the docs to point to the latest?

Reply:
it says upwards of 6.9.2 and I didn't change it from what it was.